migrations/versions/0d3fce7c5341_create_user_permission_table.py

```diff
@@ -0,0 +1,77 @@
+"""create user permission table
+
+Revision ID: 0d3fce7c5341
+Revises: 056d1b302e38
+Create Date: 2022-07-22 00:20:54.155505
+
+"""
+
+from datetime import datetime
+from sqlalchemy.orm import Session
+
+import ambuda.database as db
```

Comment:
I recently learned that importing from prod code is not a best practice for Alembic. The rationale is that by depending on `db` here, our migration file depends on code that might change in the future, and it could even break entirely.

Instead, we should lock the specific model as of this point. You can grep for "Base" in `migrations` to see some examples of where I do this.